utils/mil_tools/nodes/online_bagger.py

```diff
@@ -47,7 +47,16 @@ def __init__(self):
                                              self.start_bagging)
 
         self.subscribe_loop()
-        rospy.loginfo('subscriber list: %s', self.subscriber_list)
+
+        rospy.loginfo('')
```

Comment:
This will just print out the time with no log message. I'm assuming this is not the behaviour you want. Did you mean to just put newlines before and after your messagge? If so, just do:
`rospy.loginfo('\nRemaining Failed Topics:\n%s\n', self.get_subscriber_list_string())
`